CellProfiler_pipelines/download_cellprofiler.yml

```diff
@@ -0,0 +1,27 @@
+name: cp4
+
+channels:
```

Comment:
Do you need to specify all these channels? Typically we namespace as well. (e.g. `conda-forge::numpy`)

Reply:
I think so since this is how it was formatted when it worked for previous versions of CP. I definitely noticed the different where they do not use `conda-forge::`, but I do not know which of these dependencies is installed by which channel. I will leave it as is since it does work.